README.md

```diff
@@ -64,3 +66,12 @@ Which then outputs the following to `target/bench/query.txt`:
 | select-1.edn |  61.69 ± 0.00 |  25.38 ± 0.00 | 36.30 ± 0.00 |   58.85% |
 | select-2.edn |  91.55 ± 0.00 |  37.61 ± 0.00 | 53.93 ± 0.00 |   58.91% |
 ```
+
+## License
+
+Copyright © 2022-2023 Yet Analytics, Inc.
+
+Distributed under the Apache License version 2.0.
+
+The [Apache Jena logo](https://commons.wikimedia.org/wiki/File:Apache_Jena_logo.svg) is a trademark of the Apache Software Foundation.
```

Comment:
The logo is (likely) a Derivative Work of the Apache Jena logo, which is itself licensed under the Apache License v2.0. I put what I think _sounds_ right here in attributing the logo as a trademark of the ASF, and there's no NOTICE file for the original Jena logo on the Wikimedia site, but I obviously want to make sure this is 100% correct.